packages/rancher-vsphere/rancher-vsphere-cpi/package.yaml

```diff
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/vsphere-charts.git
 subdirectory: charts/rancher-vsphere-cpi
 commit: 96687762c6642efc3650626a0738c8caa45ae7c3
-version: 101.1.0
```

Comment:
I think if we're bumping a major version we would want to reset the minor version, right? So it would be `102.0.0`

Other reviewers can correct me if I'm wrong, but that was my understanding of the process

Reply:
Thanks! I'll use `make remove` from now on, that seems a lot faster. I ran these

```
make remove CHART=rancher-vsphere-csi VERSION=101.1.0+up2.6.2-rancher1
make remove CHART=rancher-vsphere-cpi VERSION=101.1.0+up1.4.1
```